int64[] value
---
bool success
string message
